charts/spin-operator/README.md

```diff
@@ -17,7 +17,7 @@ Prior to installing the chart, you'll need to ensure the following:
   <!-- TODO: templatize with release version corresponding to chart's appVersion -->
 
   ```console
-  $ kubectl apply -f https://github.com/spinkube/spin-operator/releases/download/v0.1.0-rc.1/spin-operator.crds.yaml
+  $ kubectl create -f https://github.com/spinkube/spin-operator/releases/download/v0.1.0-rc.1/spin-operator.crds.yaml
```

Comment:
I think these should also be apply in case folks forgot they already installed the crds. Wdyt?